Modules/xASL_module_ASL.m

```diff
@@ -187,6 +187,22 @@
 	x.Q.NumberEchoTimes = 1;
 end
 
+%% D6. DeltaM parsing - check if all/some volumes are deltams
```

Comment:
We could add the D6 in the header description where the other sections are listed, too 😃

Reply:
Good point - added. I though about adding that already previously, and I knew that being lazy doesn't pay off and somebody will find out :D